build.sbt

```diff
@@ -32,7 +32,7 @@ lazy val core = libraryProject("core")
       scalaReflect(scalaOrganization.value, scalaVersion.value) % "provided",
       scodecBits,
       scalaCompiler(scalaOrganization.value, scalaVersion.value) % "provided"
-    ),
+    )
```

Comment:
MY BEAUTIFUL TRAILING COMMAS!!!

(I like these, because you can add to the list without bringing into the diff things that don't change.  But I'm not going to die on this hill.)

Reply:
I support re-adding these commas! :)